perl.h

```diff
@@ -1224,18 +1224,18 @@ Use L</UV> to declare variables of the maximum usable size on this platform.
 
 /* byte-swapping functions for big-/little-endian conversion */
 # define _swab_16_(x) ((U16)( \
-         (((U16)(x) & UINT16_C(0x00ff)) << 8) | \
-         (((U16)(x) & UINT16_C(0xff00)) >> 8) ))
+         (((U16)(U8) (x)                   ) << 8) |  \
+         (((U16)     (x) & UINT16_C(0xff00)) >> 8) ))
```

Comment:
Actually it is a fact that it would be more consistent; it isn't just your opinion.

The real question is does the gain in performance outweigh the inconsistency.  You're saying No, and I'm saying I am willing to take your word on that.  I have pushed the rest of the commits to blead, leaving these open for a while longer to see if others wish to comment.  If none do, I'll just abandon these

Reply:
no responses, so closing this ticket without further changes